From b1584fb91fd132f5179e5de5af5913f02b38c802 Mon Sep 17 00:00:00 2001
From: Michael W <baddaemon87@gmail.com>
Date: Mon, 1 Jan 2018 13:43:00 +0100
Subject: [PATCH 052/249] wiki: Update translation contributors

* Update the list of proofreaders for the various languages
* Due to automatic generation we now have quotes around the names -
  which is not affecting the output
* Add an own section for global proofreaders
* Add a little explanation for managers and global proofreaders
* Make use of liquid 4 syntax - remove a lot of spaces in the file

Change-Id: I756a65a1a883589ff76c53b058c3de08ca8ca030
---
 _data/translators.yml | 119 +++++++++++++++++++++++++-----------------
 pages/contributors.md |  41 +++++++++++----
 2 files changed, 101 insertions(+), 59 deletions(-)

diff --git a/_data/translators.yml b/_data/translators.yml
index 1843a36..9f3b42b 100644
--- a/_data/translators.yml
+++ b/_data/translators.yml
@@ -6,101 +6,124 @@ managers:
     - name: Michael Bestas
       nick: mikeioannina
     - name: Michael W
-      nick: BadDaemon
+      nick: baddaemon87
     - name: Ricardo Cerqueira
       nick: arcee
-    - name: Simon Shields
-      nick: forkbomb
     - name: Vladislav Koldobskiy
       nick: NeverGone
+    - name: zifnab06
+      nick: zifnab06
+global_proofreaders:
+    - name: Danny Baumann
+      nick: maniac103
+    - name: Eddy Witkamp
+      nick: eddywitkamp
+    - name: Paul Keith
+      nick: javelinanddart
+    - name: Simon Shields
+      nick: forkbomb
 languages:
+    - name: Afrikaans
+      proofreaders: ['Corrie Meyer (isimobile)', 'Frans van Tonder (frans2017)']
+    - name: Albanian
+      proofreaders: ['Albi Alikaj (eurohunter)']
     - name: Arpitan
-      proofreaders: [Alexandre Raymond (Alekcxjo)]
+      proofreaders: ['Alexandre Raymond (Alekcxjo)']
     - name: Asturian
-      proofreaders: [ivarela (malditoastur)]
+      proofreaders: ['enolp', 'ivarela (malditoastur)']
     - name: Basque
-      proofreaders: [Osoitz]
+      proofreaders: ['Iñigo Zendegi (izendegi)', 'Osoitz']
     - name: Bengali
-      proofreaders: [Anik khan (Anik1199)]
+      proofreaders: ['Anik khan (Anik1199)']
     - name: Bulgarian
-      proofreaders: [Hristo Dipchikov (hristodipchikov)]
+      proofreaders: ['Hristo Dipchikov (hristodipchikov)']
     - name: Catalan
-      proofreaders: [Arnau]
+      proofreaders: ['Arnau', 'Jaime Muñoz Martín (jmmartin_5)']
     - name: Chinese Simplified
-      proofreaders: [YFdyh000, 陈少举 (chenshaoju), Yun Liu (liuyun97)]
+      proofreaders: ['YFdyh000', 'Yun Liu (liuyun97)', '陈少举 (chenshaoju)']
     - name: Chinese Traditional
-      proofreaders: [阿捷 aJay (boxajay)]
+      proofreaders: ['阿捷 aJay (boxajay)']
     - name: Croatian
-      proofreaders: [David Boca (davidboca)]
+      proofreaders: ['David Boca (davidboca)']
     - name: Czech
-      proofreaders: [Ondřej Zima (Keltek)]
+      proofreaders: ['Ondřej Zima (Keltek)']
     - name: Danish
-      proofreaders: [Jacob_DK]
+      proofreaders: ['Jacob_DK']
     - name: Dutch
-      proofreaders: [Eddy Witkamp (eddywitkamp), Peter Buijsman (Dawnes)]
+      proofreaders: ['Eddy Witkamp (eddywitkamp)', 'Peter Buijsman (Dawnes)']
     - name: English, Australia
-      proofreaders: [Simon Shields (forkbomb)]
+      proofreaders: ['Simon Shields (forkbomb)', 'xwack']
+    - name: English, India
+      proofreaders: ['xwack']
     - name: English, United Kingdom
-      proofreaders: [Harry Youd (harryyoud), jackb-p]
+      proofreaders: ['Harry Youd (harryyoud)', 'jackb-p']
     - name: Finnish
-      proofreaders: [Niko Koivu (niko0o)]
+      proofreaders: ['Niko Koivu (niko0o)']
     - name: French
-      proofreaders: [Adrian DC (RadianDC)]
+      proofreaders: ['Adrian DC (RadianDC)']
     - name: Galician
-      proofreaders: [mandruis (manuiglesias96)]
+      proofreaders: ['mandruis (manuiglesias96)']
     - name: German
-      proofreaders: [Danny Baumann (maniac103), Kim Le (elektroschmock78), Michael W (BadDaemon)]
+      proofreaders: ['Danny Baumann (maniac103)', 'Kim Le (elektroschmock78)', 'Michael W (BadDaemon)']
     - name: Greek
-      proofreaders: [Michael Bestas (mikeioannina)]
+      proofreaders: ['Michael Bestas (mikeioannina)']
     - name: Hebrew
-      proofreaders: [Nir Bruderman (nbruderman), ‫אלקנה בירדוגו‬‎ (ttv200)]
+      proofreaders: ['Nir Bruderman (nbruderman)', 'אלקנה בירדוגו‬‎ (ttv200)‎']
     - name: Hungarian
-      proofreaders: [Tamás Tóth (tothtamassyman)]
+      proofreaders: ['Tamás Tóth (tothtamassyman)']
     - name: Indonesian
-      proofreaders: [nikfrmn, Baguz Ach (eggoez), tony®ananta (qjohn)]
+      proofreaders: ['adul faker (adulfaker)', 'Baguz Ach (eggoez)', 'Miya (Miyaku)', 'nikfrmn', 'tony®ananta (qjohn)']
     - name: Italian
-      proofreaders: [Joey Rizzoli (linuxx)]
+      proofreaders: ['Joey Rizzoli (linuxx)']
     - name: Japanese
-      proofreaders: [AndroPlus (Noumi)]
+      proofreaders: ['AndroPlus (Noumi)', 'BLACK (blackzigong)']
+    - name: Kabyle
+      proofreaders: ['Belkacem Mohammed (belkacem77)']
     - name: Kashubian
-      proofreaders: [Krzysztof Mesek (mesiu84)]
+      proofreaders: ['Krzysztof Mesek (mesiu84)']
     - name: Korean
-      proofreaders: [AHN Jonggunn (pgrules7)]
+      proofreaders: ['pgrules7']
     - name: Luxembourgish
-      proofreaders: [Michel Weimerskirch (michel_weimerskirch), Edson Souza Morais (soued031)]
+      proofreaders: ['Edson Souza Morais (soued031)', 'Michel Weimerskirch (michel_weimerskirch)']
     - name: Nepali
-      proofreaders: [avseq]
+      proofreaders: ['avseq']
     - name: Norwegian Bokmal
-      proofreaders: [Vegard Fladby (Allram), Syver Stensholt (sssandum)]
+      proofreaders: ['Syver Stensholt (sssandum)', 'Vegard Fladby (Allram)']
     - name: Occitan
-      proofreaders: [sebcxjo]
+      proofreaders: ['sebcxjo']
+    - name: Pirate English
+      proofreaders: ['Paul Keith (javelinanddart)']
     - name: Polish
-      proofreaders: [Mateusz (godlesplay)]
+      proofreaders: ['Mateusz (godlesplay)']
     - name: Portuguese
-      proofreaders: [Sergio Borges (sergioborges99)]
+      proofreaders: ['Sergio Borges (sergioborges99)']
     - name: Portuguese, Brazilian
-      proofreaders: [João Paulo Just Peixoto (jpjust), Alexandre Oliveira (rockytvbr)]
+      proofreaders: ['Alexandre Oliveira (rockytvbr)', 'João Paulo Just Peixoto (jpjust)']
     - name: Romanian
-      proofreaders: [Pop Alexandru Radu (minealex2244)]
+      proofreaders: ['Pop Alexandru Radu (minealex2244)']
     - name: Russian
-      proofreaders: [Vladislav Koldobskiy (NeverGone)]
+      proofreaders: ['Vladislav Koldobskiy (NeverGone)']
+    - name: Sardinian
+      proofreaders: ['tzium']
+    - name: Scottish Gaelic
+      proofreaders: ['akerbeltz']
     - name: Serbian (Cyrillic)
-      proofreaders: [Драган (WD40), n1kolaa]
+      proofreaders: ['n1kolaa', 'Rancher', 'Драган (WD40)']
     - name: Serbian (Latin)
-      proofreaders: [n1kolaa]
+      proofreaders: ['n1kolaa', 'Rancher', 'Драган (WD40)']
     - name: Slovak
-      proofreaders: [Róbert Barát (rbarat)]
+      proofreaders: ['Róbert Barát (rbarat)']
     - name: Slovenian
-      proofreaders: [Zoran (totalyfake666)]
+      proofreaders: ['Zoran (totalyfake666)']
     - name: Spanish
-      proofreaders: [tio_mauri (rodrigomaureiracontreras), Franco (franco999)]
+      proofreaders: ['Franco (franco999)', 'tiomauri (rodrigomaureiracontreras)']
     - name: Spanish, United States
-      proofreaders: [tio_mauri (rodrigomaureiracontreras), Franco (franco999)]
+      proofreaders: ['Franco (franco999)', 'tiomauri (rodrigomaureiracontreras)']
     - name: Swedish
-      proofreaders: [Simon Bäck (backsimon)]
+      proofreaders: ['Simon Bäck (backsimon)']
     - name: Turkish
-      proofreaders: [Burak Dede (TheNexus), Sinan Serbestoğlu (serbestoglusinan)]
+      proofreaders: ['Burak Dede (TheNexus)', 'Cenkay KESER (Cenkay)', 'Mevlüt TOPÇU (xboxhunter84)', 'Sinan Serbestoğlu (serbestoglusinan)']
     - name: Ukrainian
-      proofreaders: [Aleksander Belilovets (Aleksander_Belilovets), Роман Стащак (stashchak90)]
+      proofreaders: ['Aleksander Belilovets (Aleksander_Belilovets)', 'Mykola Ronik (Mantikor)', 'Роман Стащак (stashchak90)']
     - name: Vietnamese
-      proofreaders: [blueocean2308]
+      proofreaders: ['blueocean2308']
diff --git a/pages/contributors.md b/pages/contributors.md
index 2473098..8cb8b7d 100644
--- a/pages/contributors.md
+++ b/pages/contributors.md
@@ -95,37 +95,56 @@ pathways that connect to potential contributors to teams that have contribution
 <tr><th>Device</th><th>Maintainer(s)</th></tr>
 </thead>
 <tbody>
-{% for device in sorted %}
-{% assign numMaintainers = device.maintainers | size %}
-{% if device.current_branch != version or numMaintainers == 0 %}
-{% continue %}
-{% endif %}
+{%- for device in sorted %}
+{%- assign numMaintainers = device.maintainers | size %}
+{%- if device.current_branch != version or numMaintainers == 0 %}
+{%- continue %}
+{%- endif %}
 <tr><td><b><a href="{{ "/devices/" | append: device.codename | relative_url }}">{{ device.vendor }} {{ device.name }} ({{ device.codename }})</a></b></td><td>{{ device.maintainers | join: ', ' }}</td></tr>
-{% endfor %}
+{%- endfor %}
 </tbody>
 </table>
-{% endfor %}
+{%- endfor %}
 
 ## Translations
 
+We use [crowdin](https://crowdin.com/profile/LineageOS) as our translation system. To see how that works and how to contribute, click [here](/translate-howto.html)
+
 ### Translations managers
 
+Translation managers are responsible for the general project. They approve new memberships and are responsible for keeping LineageOS and the translations in sync.
+
 <table>
 <thead>
 <tr><th>Name</th><th>Nickname</th></tr>
 </thead>
 <tbody>
-{% for el in site.data.translators.managers %}
+{%- for el in site.data.translators.managers %}
 <tr><td>{{ el.name }}</td><td>{{ el.nick }}</td></tr>
-{% endfor %}
+{%- endfor %}
+</tbody>
+</table>
+
+### Global proofreaders
+
+Global proofreaders can help out across all languages.
+
+<table>
+<thead>
+<tr><th>Name</th><th>Nickname</th></tr>
+</thead>
+<tbody>
+{%- for el in site.data.translators.global_proofreaders %}
+<tr><td>{{ el.name }}</td><td>{{ el.nick }}</td></tr>
+{%- endfor %}
 </tbody>
 </table>
 
 ### Translations proofreaders
 
-{% for lang in site.data.translators.languages %}
+{%- for lang in site.data.translators.languages %}
 * __{{ lang.name }}__: {{ lang.proofreaders | join: ', ' }}
-{% endfor %}
+{%- endfor %}
 
 ## Other projects
 
-- 
2.17.1

